configs/sngan_proj/sngan_proj_128_wReLUinplace_imagenet1k_b128x2_Glr-2e-4_Dlr-5e-5_ndisc5.py

```diff
@@ -4,16 +4,17 @@
 ]
 
 num_classes = 1000
+init_cfg = dict(type='studio')
```

Comment:
You may follow the rule of naming a [config file](https://mmgeneration.readthedocs.io/en/latest/tutorials/config.html). For example, this config should be named as `sagan_proj_wReLUinplace_Glr-2e-4_Dlr-5r-5_ndisc5_imgenet1k_b128x2`.

Please revise the name of other config files and change the related readme links.